src/main/java/io/github/mapepire_ibmi/Pool.java

```diff
@@ -298,7 +310,7 @@ public CompletableFuture<SqlJob> popJob()
             InterruptedException, ExecutionException, URISyntaxException, SQLException, UnknownServerException {
         // TODO: dead code: what to do with it?
```

Comment:
This is related to https://github.com/Mapepire-IBMi/mapepire-java/issues/43 preventing some of this code from executing. This will be fixed in a separate PR